### PR TITLE
Feature/registry admin poc

### PR DESCRIPTION
**Pull request by StyleT**
https://take.ms/wm6z4

### CODE REVIEW

File: install_all.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const spawnSync = require('child_process').spawnSync;
 const commands = [
     { command: 'cd ./ilc/ && npm i', name: 'ilc' },
     { command: 'cd ./registry/ && npm i', name: 'registry' },
+    { command: 'cd ./registry/client && npm i', name: 'registry' },
```
Comment (nightnei) [inline install_all.js:7]: let's rename to eg `name: 'registry_admin'`

Comment (StyleT) [reply install_all.js]: Renamed to registry:ui to be consistent with `start_all.js`

Comment (StyleT) [reply install_all.js]: Renamed to registry:ui to be consistent with start_all.js

File: registry/tests/test.spec.ts [hunk 2]
```diff
@@ -3,6 +3,6 @@ import { request, expect } from './common';
 describe('Tests /', () => {
     it('Hello world', async () => {
         const response = await request.get('/');
-        expect(response.text).to.be.equal('Hello! This is Micro Fragments registry service.');
+        expect(response.text).to.contain(' <title>React Admin</title>');
```
Comment (oleh-momot) [inline registry/tests/test.spec.ts:6]: I guess, space is not necessary here (' <title...')

Comment (StyleT) [reply registry/tests/test.spec.ts]: updated

**Pull request merged**